juriscraper/opinions/united_states/state/nd.py

```diff
@@ -90,14 +113,12 @@ def clean_name(self, name: str) -> tuple[str, str]:
         :param name: raw case name
         :return: cleaned name and extra_docket numbers
         """
-        other_dockets = ""
         if "(consolidated w/" in name:
-            other_dockets = ",".join(re.findall(r"\d{8}", name))
```

Comment:
you updated the function but didnt update the docstring.  it still says its returning extra docket numbers.  But shouldnt those extra docket numbers be included in the final docket numbers?  Can you create a second example html that captures this edge cases in the html so I can see how it is processed